console/src/telemetry/Actions.ts

```diff
@@ -12,7 +13,15 @@ const SET_CONSOLE_OPTS = 'Telemetry/SET_CONSOLE_OPTS';
 const SET_NOTIFICATION_SHOWN = 'Telemetry/SET_NOTIFICATION_SHOWN';
 const SET_HASURA_UUID = 'Telemetry/SET_HASURA_UUID';
 
-const setConsoleOptsInDB = (opts, successCb, errorCb) => (
+type Opts = {
```

Comment:
Instead of using `Partial` for `Opts`, you can do:
```ts
type Opts = {
  telemetryNotificationShown?: boolean;
  disablePreReleaseUpdateNotifications?: boolean;
};
```
`Opts` type is used always with `Partial` anyway.
Also, maybe we can rename it to `Options`?